docs/opcodes/55/berlin.mdx

```diff
@@ -1,5 +1,10 @@
 ## Gas
 
+### Definitions:
+- **value**: The value of the storage slot after executing the opcode SSTORE.
+- **current_value**: The value of the storage slot before executing the opcode SSTORE.
```

Comment:
The description for the "estimate" section is "current value in storage". Can we stay consistent? Either modify this one or the one in the other section. Though I prefer mine ;-)

Reply:
I changed it